python/plugins/per-project-location/location/per_project_location.py

```diff
@@ -0,0 +1,77 @@
+import os
+import sys
+import platform
+import functools
+import tempfile
+import logging
+
+import ftrack_api
+from ftrack_api.entity.location import Location
+from ftrack_api.accessor.disk import DiskAccessor
+from ftrack_api.structure.standard import StandardStructure
+
+
+logger = logging.getLogger(
```

Comment:
No line break needed here, please run Black prior to commit:

`black -l 79 --skip-string-normalization .`

Reply:
done, thanks.